products/dotnet/get-started.adoc

```diff
@@ -42,7 +44,7 @@ Start _Red Hat Subscription Manager_ using the _System Tools_ group of the _Appl
 . Select _Repositories_ from the _System_ menu of the subscription manager.
 . In the list of repositories, check the _Enabled_ column for _rhel-7-server-dotnet-rpms_. Note: After clicking, it might take several seconds for the checkmark to appear in the enabled column.
 
-If you don’t see any .NET Core repositories in the list, your subscription might not include it. See <<troubleshooting,Troubleshooting and FAQ>> for more information. +
+If you don’t see any .NET repositories in the list, your subscription might not include it. See <<troubleshooting,Troubleshooting and FAQ>> for more information. +
 
 
 ### Using subscription-manager from the command line
```

Comment:
Please revise the last sentence in line 96 to read: "Each time you log in or start a new terminal session, you will need to run `scl enable`."

Question: Is this where we need to add the following text from Don?
Every time you open a bash session in your VM, you’ll need to run the command shown above. Not only is this a nuisance, but it won’t work if you wish to debug your code from within Visual Studio. To avoid this issue, add the following line to the file ~/.bashrc in your VM:

source scl_source enable rh-dotnetcore10